docs/guides/misc/setting-static-ip.md

```diff
@@ -0,0 +1,78 @@
+[DRAFT]
+
+When you plug your device into the network, you will be assigned an IP address from the router. For most use cases, this is all that is needed, but for servers it is strongly recommended that the IP address is made static.
+
+This means ensuring that no matter how many times the device is rebooted / disconnected and reconnected it will _always_ get the same IP address.
+
+Depending on your operating system / device. There are many ways to achieve this, we will list a few here.
+
+# Raspbian / Raspberry Pi OS
+
+First, lets make take some notes about our current network settings with the following command:
+
+```bash
+ip r | grep default
+```
+
+This will spit out something along the lines of:
+
+```text
+default via 192.168.1.254 dev eth0 proto dhcp src 192.168.1.80 metric 202`
+```
+
+The first IP address here (`192.168.1.254`) is the IP address of our router. The second one (`192.168.1.80`) is the IP address that the router has assigned our Raspberry pi. Depending on your requirements, you can either choose to keep this IP or change it entirely. Just make sure that the IP address you choose is not already in use.
+
+We also need to know the DNS server that has been set. to do this run the following command:
+
+```bash
+cat /etc/resolv.conf
+```
+
+Which will output something along the lines of:
+
+```
+# Generated by resolvconf
+nameserver 192.168.1.254
+```
+
+We are interested in the IP address next to `nameserver`.
+
+Now that you have the above IP addresses noted down, it's time to modify `dhcpcd.conf` to set it as static. To do so, first run this command:
+
+```bash
+sudo nano /etc/dhcpcd.conf
+```
+
+This will bring up a text editor (`nano`) - into which you can enter your desired settings.
+
+One other thing we will need to know is the interface. If you are plugged into the network via a cable (strongly recommended), then you will want `eth0`, or if you have connected via WiFi, then you will want `wlan0`
```

Comment:
This would probably fit as well better into the upper section. The default route contains the primary interface (www) interface name, which can be have these `enp3s0`/`wlp3s0` style "predictable" names: https://www.freedesktop.org/software/systemd/man/systemd.net-naming-scheme.html

So before users are confused seeing these, while the guide tells something about `eth0`/`wlan0`, it may be better to refer to the default route output, using what it prints out?

Also above it sounds like it can be chosen at this point, while of course when the default route was via Ethernet before, there are likely no WPA settings applied to use WiFi OOTB, so in any case it should be simply the interface of the default route.